### PR TITLE
Update types.ts

### PR DESCRIPTION
**Pull request by CryptoUncleSam**

### CODE REVIEW

**Comment by JoshuaSchmidt-OpenSea** [comment]
@CryptoUncleSam Thanks for adding this. I plan to review this and make any changes necessary tomorrow (or Monday). Can  you also add at least one test for this? At a minimum an assertion to the current `getCollection` integration test.

File: src/types.ts [hunk 1]
```diff
@@ -304,7 +304,72 @@ interface NumericalTraitStats {
...
+    thirty_day_difference: number;
+    total_volume: number;
+    total_sales: number;
+    total_supply: number;
+    count: number;
+    num_owners: number;
+    average_price: number;
+    num_reports: number;
+    market_cap: number;
+    floor_price: number;
```
Comment (JoshuaSchmidt-OpenSea) [inline src/types.ts:371]: `floor_price` could be null or blank

**Comment by JoshuaSchmidt-OpenSea** [comment]
I am merging this to a branch in the repo so I can clone it locally and edit some the types to also accept undefined/null
